src/stories/4-elements/text/text.stories.tsx

```diff
@@ -11,8 +11,17 @@ const meta: Meta = {
 				<>
 					<Title>Text</Title>
 					<p>
-						This component renders a <code>Text</code> component provided by the Design System within a
-						Frontend Engine generated form.
+						This component renders a <code>Typography</code> component provided by the Design System within
+						a Frontend Engine generated form. For backward compatibility, it retains the original HTML
+						element and weight of <code>V2_Text</code>. Please refer to the{" "}
+						<a
```

Comment:
there are now duplicate lines, can remove the second one as it's redundant

![Screenshot 2025-04-30 at 9 09 42 AM](https://github.com/user-attachments/assets/13e27340-bf61-4831-b875-25c979404a21)